doc/release/RELEASE-PROTOCOL.md

```diff
@@ -132,17 +147,17 @@ Each new feature should reside in its own branch, which can be pushed to the cen
 
 Once `main` has acquired enough features for a release (or a predetermined release date is approaching), you run the `[Release] Init` GitHub workflow that creates a new release branch with a version number for the release. The release version number for release branches includes only the major and minor versions.
 
-When the release branch is ready for release, the `[Release] Deploy` workflow should be run for each environment, such as sandbox and production. This workflow will create two Git tags, each with a version number.
-This workflow will also initiate the Deploy to AWS workflow and pass it the necessary parameters. If you need to clear the custom OpenSearch indexes and tenplates during deployment, you need to select the `Clear the custom OpenSearch indexes and templates` checkbox.
+When the release branch is ready for release, the `[Release] Deploy` workflow should be run for each environment, such as Sandbox and Production. This workflow will create two Git tags, each with a version number.
```

Comment:
I suggest improving this paragraph a little bit.
What do you think about my version?

_When the release branch is ready, run the [Release] Deploy workflow for each target environment (e.g., Sandbox, Production). This workflow automatically creates a Git tag that includes an environment-specific prefix along with the appropriate version number._